chat-widget/src/components/startchaterrorpanestateful/interfaces/IStartChatErrorPaneClassNames.ts

```diff
@@ -0,0 +1,5 @@
+export interface IStartChatErrorPaneClassNames {
```

Comment:
Please update dev guide on the new interfaces

Reply:
Added dev docs